problem_builder/public/js/mentoring_with_steps.js

```diff
@@ -364,11 +365,22 @@ function MentoringWithStepsBlock(runtime, element) {
         });
     }
 
+    function buildInteractionData() {
```

Comment:
Shall we add a short description of what is this data for?

Reply:
@tomaszgy I can't think of a better description rather than "this data is used in `xblock.interaction`" event". The thing is - the mechanism is pretty generic and actual use of data is left for XBlock runtime to determine - and the runtime isn't here. So, in my opinion, it's not a good idea to describe how this data is used by the runtime (since there might be different uses across runtimes).

Reply:
Thanks - the current comments are very good :)